README.md

```diff
@@ -1 +1,24 @@
 # Operator Marketplace
+
+## Deploying the Marketplace Operator
+### Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Be logged into an OpenShift cluster as a user with the RBAC permissions required to run this operator. These RBAC permissions are currently undefined, so log in as a user with the Cluster Admin Role.
+2. Have created the CatalogSourceConfig custom resource defined within the [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) project.
+3. Have authorization to pull the Marketplace Operator Image from quay.io/redhat/marketplace, you can make a request to join the “RedHat” organization by following the steps outlined [here](https://mojo.redhat.com/docs/DOC-1167707)
+4. Update the deploy/marketplace-pull-secret.yaml file with your secret.
+
```

Comment:
Maybe add a separate section on how to pull from a private repo using pull-secret?

Reply:
Given that the image is now public, this won't be required now. It would be nice if you can test and confirm that.

Reply:
Tested and confirmed that pull-secret is no longer needed.